### PR TITLE
Updated Sprint

### PR DESCRIPTION
**Pull request by fbernhart**
sprint

**Issue:** #3169 
**Alexa rank:** -

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json` --> (wasn't necessary)
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I took the already used SVG.
I used Inkscape to fix size and center
I used https://jakearchibald.github.io/svgomg/ to optimize the resulting SVG

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
Seems Sprint merged with T-Mobile in April 2020. @simple-icons/maintainers should we mark this for removal?

**Comment by ericcornelissen** [comment]
Not sure, judging by [the website](https://www.sprint.com/en/login.html) it seems to still be a somewhat separate brand :thinking:

**Comment by adamrusted** [comment]
> Not sure, judging by [the website](https://www.sprint.com/en/login.html) it seems to still be a somewhat separate brand 🤔

Weird - though the moment I click a link in the navigation it takes me straight to T-Mobile.

**Comment by ericcornelissen** [comment]
Didn't click beyond that page :sweat_smile: But you're right. Personally I would just keep it for now and see what happens in the future.

**Comment by fbernhart** [comment]
I would as well keep it for now.

File: icons/sprint.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Sprint icon</title><path d="M2.1514 24c5.4942-1.8084 13.4599-6.0213 19-9.784.4651-.3141.8272-.6072.8272-1.1183 0-.8892-1.6554-2.0804-2.4585-2.5555C15.16 14.9992 6.4844 21.6785 2.1504 24m5.0971-3.7858c3.3887-2.3045 8.3117-6.6394 9.977-8.9979.3932-.554.6832-.8761.6832-1.4563 0-.9522-1.1282-1.6503-1.9404-2.1354-1.9464 4.511-6.3173 10.1271-8.7198 12.5896m2.5915-3.5287c2.3445-2.9676 4.189-6.4554 4.189-8.9599 0-1.3083-.9533-2.2225-2.1965-2.9166.024 2.7216-.9342 9.1029-1.9925 11.8765m.4962-2.8626c.438-2.8476.18-6.2863-.3791-8.4438-.304-1.1742-1.3803-2.5005-2.3765-2.9136 1.3053 3.6438 2.4265 8.8339 2.7556 11.3574M9.37 9.827C8.3437 7.0985 5.7852 1.4103 2.0214 0 4.472 2.6606 8.1057 7.9367 9.37 9.827"/></svg>
```
Comment (adamrusted) [inline icons/sprint.svg:1]: There seem to be some differences between PR (red) and [source](https://www.sprint.com/content/dam/sprint/us/en/design-elements/images/t-mobile-sprint-logo.svg) (black) - could you rebuild using the 'latest' logo?

image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/sprint.svg (hunk 1 above)
Comment (fbernhart) [reply icons/sprint.svg:1]: @adamrusted Fixed 👍🏼

**Review by adamrusted — APPROVED** [review]
LGTM!

**Pull request merged**